homeassistant/components/screenlogic/config_flow.py

```diff
@@ -0,0 +1,247 @@
+"""Config flow for ScreenLogic."""
+import logging
```

Comment:
This one is going to fail the CI since we require 100% coverage for config flows.

Reply:
`python3 -m script.scaffold config_flow` can help generate the base tests

Reply:
Comit [b52ded3](https://github.com/home-assistant/core/pull/47933/commits/b52ded3c135330202d72185ac6f4f4c556dd68bb) should take care of this.